sensor/extrinsic_reflector_based_calibrator/src/extrinsic_reflector_based_calibrator.cpp

```diff
@@ -1150,14 +1183,27 @@ void ExtrinsicReflectorBasedCalibrator::calibrateSensors()
 
     // sin(a-b) = sin(a)*cos(b) - cos(a)*sin(b)
     // cos(a-b) = cos(a)*cos(b) + sin(a)*sin(b)
-    // a = lidar
-    // b = radar
+    // a = lidar, b = radar
     double delta_angle_sin = lidar_transformed_sin * radar_cos - lidar_transformed_cos * radar_sin;
     double delta_angle_cos = lidar_transformed_cos * radar_cos + lidar_transformed_sin * radar_sin;
     delta_sin_sum += delta_angle_sin;
     delta_cos_sum += delta_angle_cos;
+
+    // logging
+    RCLCPP_INFO_STREAM(this->get_logger(), "lidar_estimation:\n" << lidar_estimation.matrix());
+    RCLCPP_INFO_STREAM(
+      this->get_logger(), "lidar_transformed_estimation:\n"
+                            << lidar_transformed_estimation.matrix());
+    RCLCPP_INFO_STREAM(
+      this->get_logger(), "radar_estimation_rcs:\n"
+                            << radar_estimation_rcs.matrix());
   }
+}
 
+void ExtrinsicReflectorBasedCalibrator::estimateTransformationSVD(
+  pcl::PointCloud<PointType>::Ptr lidar_points_pcs,
```

Comment:
The SVD and the 2D-rotation-only are completely different methods.
There is no reason for the 2D-rotation-only to be inside the SVD method...

Reply:
It is a naming issue :(
I changed the `estimateTransformationSVD ` to `estimateTransformation`.
https://github.com/tier4/CalibrationTools/pull/145/commits/82d940da6f5dfdebeea22fd0649d4526e63a8c7f